src/component/CrudOnePage/CrudOnePage.AppToolbar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex, Spacer } from "@chakra-ui/react"
+import { Button } from "../Button"
+import { PageHeader } from "../PageHeader"
+
+interface AppToolbarProps {
+  noCreateButton?: boolean
+  pageTitle: string
+}
+
+const pageHeader = (title: string) => {
+  return (
+    <PageHeader fontWeight={'bold'} flexDirection={'column'} justifyContent={'center'}>
+      {title}
+    </PageHeader>
+  )
+}
+
+export const AppToolbar = (props: AppToolbarProps) => {
```

Comment:
disini `showCreateButton` nya bisa dikasih default value

```suggestion
export const AppToolbar = ({
  noCreateButton = true,
  pageTitle,
}: AppToolbarProps) => {
```